docs/api-usage-graphql.rst

```diff
@@ -0,0 +1,52 @@
+############################
+Using the GraphQL API (beta)
+############################
+
+python-gitlab provides basic support for executing GraphQL queries and mutations.
+
+.. danger::
+
+   The GraphQL client is experimental and only provides basic support.
+   It does not currently support pagination, obey rate limits,
+   or attempt complex retries. You can use it to build simple queries and mutations.
+
+   It is currently unstable and its implementation may change. You can expect a more
+   mature client in one of the upcoming versions.
+
+The ``gitlab.GraphQL`` class
+==================================
+
+As with the REST client, you connect to a GitLab instance by creating a ``gitlab.GraphQL`` object:
+
+.. code-block:: python
+
+   import gitlab
+
+   # anonymous read-only access for public resources (GitLab.com)
+   gl = gitlab.GraphQL()
```

Comment:
Probably silly, but what if it was `gq` instead of `gl`?  To minimize confusion with all of our use of `gl` for the REST client.

Reply:
I was thinking about this too, hence the weird `gl_gql` in the tests :sweat_smile: yeah `gq` might be a nice one, thanks. :+1: